### ISSUE TITLE
Timeline invalid aria attributes

### ISSUE THREAD

**Issue by bddjong** [nl-design-system/denhaag#731]
### Describe the problem

The Timeline component fails it's accessibility tests due to invalid aria attributes. The `aria-expanded` attribute is not valid on a `div` element without a role. For now, the rule has been disabled in the `Timeline.spec.jsx` (line 85). 

### Describe the solution you'd like

The timeline attributes should be valid and pass the cypress a11y tests.

### Describe alternatives you've considered

Arguably, steps that are not interactable should not have the `aria-expanded` attribute, as there is no way to expand them. This would solve the invalid attribute issue. 

### Acceptance criteria

Conditions that must be satisfied to be accepted by a user, customer or other stakeholder.

- [ ] Timeline component passes a11y tests without ignoring the `aria-allowed-attr` rule
